packages/twenty-server/src/modules/calendar/calendar-event-import-manager/drivers/utils/sanitize.ts

```diff
@@ -0,0 +1,24 @@
+export const sanitizeObject = <T extends Record<string, any>>(
```

Comment:
Could we use a more narrow name? `sanitizeObject` and `sanitizeString` feel very generic and it feels like something that could be mis-imported/conflict with other parts of the codebase?